iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -303,14 +301,31 @@ class ConversationListViewController: ViewController, ConversationListCollection
                 self?.messageInputAccessoryView.textView.setPlaceholder(for: users, isReply: false)
                 self?.messageInputAccessoryView.updateSwipeHint(shouldPlay: true)
             }
-        } else {
-            ConversationsManager.shared.activeConversation = nil
 
-            if self.isFirstResponder {
-                self.resignFirstResponder()
+            self.firstResponderTask = Task { [weak self] in
+                await Task.sleep(seconds: 0.25)
+
+                guard !Task.isCancelled else { return }
+
+                // The input accessory view should be shown when centered on a conversation. If there's not
+                // already set as first responder, then make the VC first responder.
+                if UIResponder.firstResponder.isNil {
+                    self?.becomeFirstResponder()
+                }
```

Comment:
When the keyboard is up, the text field is actually the first responder, which is why we can't just check ourselves.